app/src/main/res/xml/autofill_service_configuration.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<autofill-service xmlns:android="http://schemas.android.com/apk/res/android">
-    <!--
-    "Maintain alphabetical order for these compatibility packages and ensure synchronization with
-    /xml-v30/autofill_service_configuration.xml.
-    -->
+<autofill-service xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:supportsInlineSuggestions="true"
```

Comment:
This property does not cause any issues on older versions, it will simply be ignored.